src/notifications/container.js

```diff
@@ -218,7 +221,8 @@ class NotificationContainer extends Component {
         } else {
             return unreadNotificationList.length === 0 ? (
                 <NoNotification title="No new notifications">
-                    ¯\_(ツ)_/¯
+                    {' '}
+                    ¯\_(ツ) _ / ¯
```

Comment:
This might mess up that little guy's formatting. Could you try to remove those spaces?